templates/mmm/abstract.html

```diff
@@ -1,11 +1,12 @@
 {% extends 'base.html' %}
 {% block content %}
-<h3>Historical and Projected Temperature + Precipitation Minimum, Mean, and Maximum</h3>
-<p>The endpoint here provides access to the temperature at surface in degrees Fahrenheit and total annual precipitation in inches for both historical and projected model data over the state of Alaska.</p>
```

Comment:
I think something might be going on with this file, it looks like some of the changes I made in my previous PR are getting overwritten back to what they were. E.g., the units on this line should be Fahrenheit and inches.